libs/code-generator/src/lib/code-generator.ts

```diff
@@ -13,10 +13,10 @@ import type { Tree } from './tree/tree'
 const promptsInternal: typeof prompts =
     typeof prompts == 'function' ? prompts : require('prompts')
 
-export type TemplateType = 'dotnet-api'
+export type Template = 'dotnet-api'
```

Comment:
Renamed from templateType to template

Reply:
Can we remember to update the docs

Reply:
That's in the separate repo?

I'll update once I add the auth stuff in